Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/CertificateManagerImpl.java

```diff
@@ -315,8 +315,9 @@ private void initTrustStore() throws CertificateManagerException {
      * @see gov.hhs.fha.nhinc.callback.openSAML.CertificateManager#getDefaultCertificate()
      */
     @Override
-    public X509Certificate getDefaultCertificate() throws CertificateManagerException {
-        PrivateKeyEntry pkEntry = getPrivateKeyEntry();
+    public X509Certificate getDefaultCertificate(String alias) throws CertificateManagerException {
```

Comment:
Can we rename this method getDefaultCertificate to something more appropriate comething along these lines getCertificateFor /getCertificateBasedOn